mod/research/set/self/research_page.rb

```diff
@@ -38,12 +38,13 @@ def right_side_tabs
       tabs["Source"] = cite_source_tab hide: !cite_mode?
       tabs["View Source"] = view_source_tab hide: hide_view_source_tab?
     end
-    tabs["Metric details"] = metric_details_tab if metric?
-    tabs['Need Help? <span class="badge badge-danger">New</span>'] =
-      nest :how_to_research, view: :content
+    tabs["Methodology"] = metric_details_tab if metric?
+    tabs["Need Help?"] = nest :how_to_research, view: :content
     static_tabs tabs, active_tab, "tabs", pane: { class: "p-3" }
   end
 
+  NEW_BADGE = '<span class="badge badge-danger">New</span>'.freeze
```

Comment:
did we want this to go anywhere?

Reply:
no, only hold on to it in case we need it again in the future